environs/imagemetadata/validation.go

```diff
@@ -33,9 +30,11 @@ func ValidateImageMetadata(params *simplestreams.MetadataLookupParams) ([]string
 			Endpoint: params.Endpoint,
 		},
 		Series: []string{params.Series},
-		Arches: params.Architectures,
 		Stream: params.Stream,
 	})
+	if len(params.Architectures) != 0 {
```

Comment:
do we need to make this conditional? is a nil slice treated differently to a non-nil but empty slice?
 Add a line note